website/docs/Capabilities.md

```diff
@@ -225,8 +225,8 @@ Running a headless browser means to run a browser instance without window or UI.
 
 ```ts
 {
-    browserName: 'chrome',
-    'goog:chromeOptions': {
+    browserName: 'chrome',   // 'chromium'
+    'goog:chromeOptions': {  // 'wdio:chromedriverOptions'
```

Comment:
let's not advocate chromedriver capabilities in this context since it is not a choice between chromeOptions or chromedriverOptions, you can totally can use both at the same time
```suggestion
    'goog:chromeOptions': {
```

Reply:
sure. removed chromedriverOption